proc_Hahn_echoph.py

```diff
@@ -8,7 +8,7 @@
 
 
 for searchstr, exp_type, nodename, postproc, label_str, slice_f in [
```

Comment:
proc_Hahn_echoph.py works with pyspecdata at a524dcad0c2c33b3e7c93a81bf405d14e782928f but once master was merged in it not longer works and says:
`Traceback (most recent call last):
  File "proc_Hahn_echoph.py", line 27, in <module>
    best_shift = hermitian_function_test(s)
  File "C:\Users\Alex Guinness\gitstuff\proc_scripts\proc_scripts\phasing.py", line 204, in hermitian_function_test
    s_foropt = s_foropt['t2':(-max_shift,max_shift)]
  File "c:\users\alex guinness\gitstuff\pyspecdata\pyspecdata\core.py", line 6294, in __getitem__
    slicedict,axesdict,errordict,unitsdict = self._parse_slices(args)
  File "c:\users\alex guinness\gitstuff\pyspecdata\pyspecdata\core.py", line 6486, in _parse_slices
    logger.debug(strm("i found",thisaxis[temp_high]))
IndexError: index 108 is out of bounds for axis 0 with size 108`

Reply:
note the triple quotes above for the traceback

Reply:
there were other problems with this script, so it doesn't run now, but I think the parse_slices issue should be fixed